p2p/debug/list.go

```diff
@@ -0,0 +1,133 @@
+package debug
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pokt-network/pocket/p2p/providers/peerstore_provider"
+	"github.com/pokt-network/pocket/p2p/types"
+	"github.com/pokt-network/pocket/p2p/utils"
+	"github.com/pokt-network/pocket/shared/modules"
+)
+
+var peerListTableHeader = []string{"Peer ID", "Pokt Address", "ServiceURL"}
+
+func PrintPeerList(bus modules.Bus, routerType RouterType) error {
+	var (
+		peers           types.PeerList
+		routerPlurality = ""
+	)
+
+	// TECHDEBT(#810, #811): use `bus.GetPeerstoreProvider()` after peerstore provider
+	// is retrievable as a proper submodule.
+	pstoreProviderModule, err := bus.GetModulesRegistry().
+		GetModule(peerstore_provider.PeerstoreProviderSubmoduleName)
+	if err != nil {
+		return fmt.Errorf("getting peerstore provider: %w", err)
+	}
+	pstoreProvider, ok := pstoreProviderModule.(peerstore_provider.PeerstoreProvider)
+	if !ok {
+		return fmt.Errorf("unknown peerstore provider type: %T", pstoreProviderModule)
+	}
```

Comment:
OPTIONAL nit: These lines are copy-pasted in a lot of places. I think there's an opportunity to create a shared helper